ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/encoding/TransactionEncoder.java

```diff
@@ -38,8 +38,8 @@ interface Encoder {
     void encode(Transaction transaction, RLPOutput output);
   }
 
-  private static final ImmutableMap<TransactionType, Encoder> TYPED_TRANSACTION_ENCODERS =
-      ImmutableMap.of(
+  private static final Map<TransactionType, Encoder> TYPED_TRANSACTION_ENCODERS =
+      Map.of(
```

Comment:
why are we getting rid of ImmutableMap?

Reply:
It was a suggestion from SonarCloud. I've reverted it back, because I am not sure why it is suggesting it.